CHANGES.rst

```diff
@@ -75,6 +75,10 @@ Changed
 - Make sure that the :command:`systemd` Tinc units correctly execute the
   :command:`tinc-down` scripts on service shutdown. [drybjed_]
 
+- Change the naming scheme of the debops.ferm_ configuration files in
+  anticipation of the upcoming changes. You might need to remove the old
+  firewall configuration files to avoid duplicate rule entries. [drybjed_]
```

Comment:
We have other similar cases like that. I would suggest to do it the same as for the `debops.ifupdown` role by registering role versions using `debops.debops_fact` and this can be used to reset to config if needed. I also see that the project is moving fast and also think hard breaks are ok, but they should be automated if possible, so the "You might need to remove the old firewall configuration files to avoid duplicate rule entries." can hopefully be removed.
For ferm, a upgrade note like "Run your site.yml playbook after upgrade" could be used and the ferm role could ensure that ssh rules are correct.